### PR TITLE
Feature/brn 11 configure ci to run tests on push

### PR DESCRIPTION
**Pull request by EEM86**
Created a simple Gradle project with a Demo Test to run it via CI.

Changed CI config to execute Gradlew test/build tasks on push and pull_request actions.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation "org.junit.jupiter:junit-jupiter-api:${junitVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-engine:${junitVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:${junitVersion}"
+
```
Comment (azarot) [inline build.gradle:31]: Please remove redundant empty lines

File: .gitignore [hunk 2]
```diff
@@ -11,7 +21,6 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
```
Comment (azarot) [inline .gitignore:14]: I don't think we need to remove this.

File: gradle.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'svydovets-bring'
```
Comment (azarot) [inline gradle.properties:1]: Please add empty line.

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```
Comment (azarot) [inline gradle/wrapper/gradle-wrapper.properties:5]: Please add an empty line.

**Review by valeriipestov — APPROVED** [review]

**Review by azarot — APPROVED** [review]

**Pull request merged**